apps/src/lib/util/timeoutList.js

```diff
@@ -0,0 +1,104 @@
+/** @file Wrap timer functions for easy cleanup of all timers. */
```

Comment:
Note: This file was moved, it just also changed enough that it looks like it was added here.